source/InnoSetup.ScriptBuilder/Directory.Build.Props

```diff
@@ -1,8 +1,8 @@
 ﻿<Project>
-    <PropertyGroup>
-        <Version>0.1.2</Version>
```

Comment:
Keep the package version untouched. It'll be updated in release

Reply:
Done